genie-web/src/main/java/com/netflix/genie/web/data/entities/JobEntity.java

```diff
@@ -123,6 +125,10 @@
     doNotUseGetters = true
 )
 @Entity
+@EntityListeners({
```

Comment:
Syntax wise we've been putting the `{` on the next line in these instances. Not a big deal though